maquetado/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Título del libro</title>
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+
+  <body>
+    <div class="book">
+      <div>
```

Comment:
I think this `div` is not needed. Remember to keep the HTML clean with no extra elements. Check how to do the badge element with pseudo-element and then you can delete this